src/app.js

```diff
@@ -38,6 +42,10 @@ class App {
 
     this.logger.info('creating app')
     const server = this.app
+      .use(async (ctx, next) => {
+        await next()
+      })
+      .use(mount('/pay', serve(path.resolve(__dirname, '../static/pay'))))
```

Comment:
In DJ's PR that adds alerts and balance adjustments he refactored how the gui serves static files. If you rebase and have a look at his commits you can host your static files in the same way